### PR TITLE
feat: some string length lemmas

### PR DESCRIPTION
**Pull request by alexjbest**
not sure if they belong here or in mathlib but they seemed widely useful

### CODE REVIEW

File: Std/Data/String/Lemmas.lean [hunk 1]
```diff
@@ -23,6 +23,16 @@ theorem ext_iff {s₁ s₂ : String} : s₁ = s₂ ↔ s₁.data = s₂.data :=
 
 @[simp] theorem mk_length (s : List Char) : (String.mk s).length = s.length := rfl
 
+@[simp] theorem length_toString (c : Char) : c.toString.length = 1 := rfl
```
Comment (digama0) [inline Std/Data/String/Lemmas.lean:26]: I think this would make more sense in the `Char` namespace, otherwise the name is too general

**Comment by semorrison** [comment]
awaiting-review

**Comment by alexjbest** [comment]
Damn I had no idea that I typo'd. This bot labelling thing is kinda annoying.

**Comment by semorrison** [comment]
Ask @digama0 for triage access. :-)

**Pull request merged**